erroranalysis/tests/test_error_report.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from erroranalysis._internal.error_report import ErrorReport
+from common_utils import (
+    create_boston_data, create_iris_data, create_cancer_data,
+    create_models_classification, create_models_regression)
+from erroranalysis._internal.error_analyzer import ModelAnalyzer
+
+
+class TestErrorReport(object):
+
+    def test_error_report_iris(self):
+        X_train, X_test, y_train, y_test, feature_names, _ = create_iris_data()
+
+        models = create_models_classification(X_train, y_train)
+
+        for model in models:
+            categorical_features = []
+            run_error_analyzer(model, X_test, y_test, feature_names,
+                               categorical_features)
+
+    def test_error_report_cancer(self):
+        X_train, X_test, y_train, y_test, feature_names, _ = \
+            create_cancer_data()
+
+        models = create_models_classification(X_train, y_train)
+
+        for model in models:
+            categorical_features = []
+            run_error_analyzer(model, X_test, y_test, feature_names,
+                               categorical_features)
+
+    def test_error_report_boston(self):
+        X_train, X_test, y_train, y_test, feature_names = \
+            create_boston_data()
+        models = create_models_regression(X_train, y_train)
+
+        for model in models:
+            categorical_features = []
+            run_error_analyzer(model, X_test, y_test, feature_names,
+                               categorical_features)
+
+
+def run_error_analyzer(model, X_test, y_test, feature_names,
+                       categorical_features):
+    model_analyzer = ModelAnalyzer(model, X_test, y_test,
+                                   feature_names,
+                                   categorical_features)
+    error_report1 = model_analyzer.create_error_report()
+    error_report2 = model_analyzer.create_error_report()
+    assert error_report1.id != error_report2.id
```

Comment:
Do you want to test that they parse to uuids?

Reply:
done!